tests/e2e/config.go

```diff
@@ -368,6 +368,7 @@ func SlashThrottleTestConfig() TestConfig {
 				IpPrefix:       "7.7.7",
 				VotingWaitTime: 20,
 				GenesisChanges: ".app_state.gov.params.voting_period = \"20s\" | " +
+					".app_state.gov.params.expedited_voting_period = \"10s\" | " +
```

Comment:
do we use this anywhere?

Reply:
It's required to be present in genesis.json, otherwise the chain will panic on start